css/subject-viewer.styl

```diff
@@ -51,6 +51,7 @@
     .text-viewer
       background-color: #404040
       color: white
+      min-width: 30em
```

Comment:
Aha, managed to reproduce it! Should be fixed by 8b663f5, which matches the width of the viewer to the width of its container.